components/config/prefs.rs

```diff
@@ -205,6 +205,7 @@ mod gen {
             dom: {
                 webgpu: {
                     /// Enable WebGPU APIs.
+                    wgpu_backend: String,
                     enabled: bool,
```

Comment:
`/// Enable WebGPU APIs.` should remain at `enabled: bool`, and `wgpu_backend: String` needs to get own doc comment.